src/sap/fhir/model/r4/FHIRFilterOperatorUtils.js

```diff
@@ -137,5 +141,101 @@ sap.ui.define(["sap/fhir/model/r4/FHIRFilterOperator", "sap/fhir/model/r4/FHIRFi
 		return sFHIRSearchPrefix;
 	};
 
+	/**
+	 * Transforms the UI5 filter operator to a FHIR valid filter prefix based on the given UI5 <code>oFilter</code>
+	 *
+	 * @param {sap.ui.model.Filter} oFilter The given filter
+	 * @returns {string} The FHIR filter prefix
+	 * @protected
+	 * @since 2.1.0
+	 */
+	FHIRFilterOperatorUtils.getFHIRFilterPrefix = function (oFilter) {
+		var sFHIRFilterPrefix;
```

Comment:
if no valid enum value is used, undefined is returned, is that what we wanna achieve? developer should be informed  via an error, that the prefix was not correct. Lets throw an error and add a test for that

Reply:
OK. I see in all other cases we simply return undefined. You're, right we can stick to the current implementation. Please forget my comment above.